model/import/Parser/TemplateDataParser.php

```diff
@@ -0,0 +1,101 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\import\Parser;
+
+use oat\oatbox\service\ConfigurableService;
+
+class TemplateDataParser extends ConfigurableService
+{
+    public function parser(array $finalHeader): array
+    {
+        
+        $sampleValues = $this->getDefaultValues($finalHeader);
+        return $sampleValues;
+    }
+
+    private function getDefaultValues($finalHeader): array
+    {
+        $correctResponse = array(
+            "[please remove] EXAMPLE 1 - item with correct answer",
+            'Select the correct response (no restriction on number of selectable choices).'
+            . 'The value in "correct_answer" should be choice identifiers (listed in the column header) '
+            . 'and not the actual text content of the choices. In this example, "choice_2" is the correct answer '
+            . 'and gives the score 1',
+            '',
+            'en-US',
+            '',
+            '',
+            'text for choice_1',
+            'text for choice_2',
+            'text for choice_3',
+            'text for choice_4',
+            '',
+            '',
+            '',
+            '',
+            'choice_2'
+        );
+        $mapResponse = array(
+            "[please remove] EXAMPLE 2 - item with partial score",
+            'Select the correct response (the choices are shuffled, at least 1 choice must be selected and '
+            . 'maximum of 2 allowed, this item uses partial scoring)',
+            '1',
+            'en-US',
+            '1',
+            '2',
+            'A',
+            'B',
+            'C',
+            'D',
+            '3',
+            '0',
+            '-2',
+            '-1',
+            ''
+        );
+        $combinationResponse = array(
+            "[please remove] EXAMPLE 3 - item with both correct answer and partial score",
+            'Select the correct response (1 single choice allowed, the correct answer is "X" and "Y", '
+            . 'this item uses partial scoring)',
+            '',
+            'en-US',
+            '',
+            '1',
+            'W',
+            'X',
+            'Y',
+            'Z',
+            '-1',
+            '1',
+            '1',
+            '0',
+            'choice_2,choice_4'
+        );
+        return array(
+            $finalHeader,
+            $correctResponse,
+            $mapResponse,
+            $combinationResponse
+        );
```

Comment:
We generally use short version of array:

```suggestion
        return [
            $finalHeader,
            $correctResponse,
            $mapResponse,
            $combinationResponse
        ];
```

Reply:
Suggested change completed